example/utils.go

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2020 AMIS Technologies
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"fmt"
+
+	"github.com/getamis/alice/crypto/tss/dkg"
+	"github.com/getamis/sirius/log"
+)
+
+const (
+	typeDKG     int = 0
+	typeSigner  int = 1
+	typeReshare int = 2
+)
+
+func writeDKGResult(id string, result *dkg.Result) error {
```

Comment:
not check in the result files

Reply:
I don't get it lol

Reply:
I mean you checked in the output files in this pull requests. They should not be checked in.